### PR TITLE
Add preview image for Tableau chart [sc-6014]

### PR DESCRIPTION
**Pull request by alyiwang**
### 🤔 Why?

provide more Tableau chart info to users

### 🤓 What?

- add chart url and preview image to crawler

### 🧪 Tested?

unit test
local run against tableau online, and examined the MCE json.

### CODE REVIEW

File: metaphor/tableau/extractor.py [hunk 1]
```diff
@@ -113,3 +124,30 @@ def parse_dashboard(self, workbook: WorkbookItem) -> None:
         )
 
         self._dashboards[workbook.id] = dashboard
+
+    def _parse_chart(self, workbook_name: str, view: ViewItem) -> Chart:
+        original_view = self._views.get(view.id)
```
Comment (mars-lan) [inline metaphor/tableau/extractor.py:129]: As discussed, let's turn this into a data URI instead

File: README.md [hunk 2]
```diff
@@ -37,17 +37,18 @@ Similarly, you can also install the package using `requirements.txt` or `pyproje
...
-| [metaphor.bigquery](metaphor/bigquery/README.md) | Schema, Description |
-| [metaphor.dbt](metaphor/dbt/README.md) | dbt models, tests, lineage |
-| [metaphor.google_directory](metaphor/google_directory/README.md) | User |
-| [metaphor.looker](metaphor/looker/README.md) | Looker views, explores, dashboards |
-| [metaphor.postgresql](metaphor/postgresql/README.md) | Schema, Description, Statistics |
-| [metaphor.redshift](metaphor/redshift/README.md) | Schema, Description, Statistics |
-| [metaphor.snowflake](metaphor/snowflake/README.md) | Schema, Description, Statistics |
-| [metaphor.snowflake.profile](metaphor/snowflake/profile/README.md) | Data Profile |
-| [metaphor.snowflake.usage](metaphor/snowflake/usage/README.md) | Usage |
+| Connector                                                          | Metadata                           |
```
Comment (mars-lan) [inline README.md:40]: Probably need to add a MD formatter to precommit to avoid this in the future.

File: metaphor/tableau/extractor.py (hunk 1 above)
Comment (alyiwang) [reply metaphor/tableau/extractor.py:129]: updated

File: README.md (hunk 2 above)
Comment (alyiwang) [reply README.md:40]: sure, adding it in next PR

**Review by mars-lan — APPROVED** [review]

**Pull request merged**